evaluation/tests/agg/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+export PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel --show-superproject-working-tree)}
+# time: print real in seconds, to simplify parsing
+
+bash="bash"
+pash="$PASH_TOP/pa.sh --r_split --dgsh_tee --r_split_batch_size 1000000 --parallel_pipelines --profile_driven"
```

Comment:
We might want to add `--assert_compiler_success` here, since all of these scripts are pure (I think) and therefore compiler should always succeed. Maybe we also want to add a flag that checks that some parallelization happened (to check that new annotations didn't break parallelization). Currently, all these tests will trivially pass (except if PaSh fails somewhere very early pathologically).

Reply:
Currently, a relatively small number of tests succeeds (less than 1/3, I believe, from a discussion with @dkarnikis .) The hope is that this will push us to the right direction (add correct cross-platform aggregators), which shouldn't be difficult.

I agree with adding the assertion.

Reply:
Ah nice!!!

Reply:
By adding the assertion flag, some tests hang ([test-7](https://github.com/binpash/pash/blob/3a841f6add4bef0f67c10b7e7e6de624316474ec/evaluation/tests/agg/test-7), [test-78](https://github.com/binpash/pash/blob/3a841f6add4bef0f67c10b7e7e6de624316474ec/evaluation/tests/agg/test-78) and possibly more).

In total, 184/197 of the tests pass (without the assertion flag)